### PR TITLE
Timestamps

### PR DESCRIPTION
**Pull request by MihranGuyumjyan**

### CODE REVIEW

File: models/history.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Schema, model } from "mongoose";
+
+const HistorySchema = new Schema({
+  userId: { type: String, required: false},
+  lastLogin: { type: String, required: false },
+});
```
Comment (ArsenAvagyan) [inline models/history.js:8]: @MihranGuyumjyan please add timetsamps here

File: models/users.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
-  email: { type: String, unique: true },
-  password: { type: String, required: true },
-});
+const UserSchema = new Schema(
+  {
+    firstName: { type: String, required: true },
+    lastName: { type: String, required: true },
+    email: { type: String, unique: true },
+    password: { type: String, required: true },
+    age: { type: Number, required: false },
```
Comment (ArsenAvagyan) [inline models/users.js]: no need required: false for age field

File: resolvers/index.js [hunk 3]
```diff
@@ -22,6 +23,22 @@ export const resolvers = {
       const isValidPassword = bcrypt.compare(password, existedUser.password);
       if (!isValidPassword) throw new ApolloError("incorrect password");
 
+      const userHistory = await History.findOne({ userId: existedUser._id });
+      const date = new Date();
```
Comment (ArsenAvagyan) [inline resolvers/index.js]: please add toISOString() to date variable instead of line 31 and 37

Comment (MihranGuyumjyan) [reply resolvers/index.js]: fixed please check

File: models/users.js (hunk 2 above)
Comment (MihranGuyumjyan) [reply models/users.js]: fixed please check

File: models/history.js (hunk 1 above)
Comment (MihranGuyumjyan) [reply models/history.js:8]: added please check

**Pull request merged**